.github/workflows/build-images-ci.yaml

```diff
@@ -223,6 +224,11 @@ jobs:
       - name: CI Build ${{ matrix.name }}
         uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
         id: docker_build_ci
+        if: |
+          ${{ always() &&
+              (matrix.require-dir == '' ||
+               hashFiles(matrix.require-dir))
```

Comment:
Suspecting that we can't pass variable content into `hashFiles()` here. Test PR　workflow run:　https://github.com/cilium/cilium/actions/runs/13802755624/job/38608035678?pr=38142#step:20:181　

Reply:
:heavy_check_mark: resolved.